try:
    file = open('test.txt', 'r')
except Exception as e:
    print("Ocurrio un error, se atropo un excepci√≥n.")